autofs-5.1.0-beta1 - fix map format init in lookup_init()

From: Ian Kent <ikent@redhat.com>

is_format_amd is not initialized in modules/lookup_ldap.c:lookup_init()
which can cause it to be incorrect leading to a failure to locate a
base dn.
---

autofs-5.1.0-beta1 - fix map format init in lookup_init()

From: Ian Kent <ikent@redhat.com>

is_format_amd is not initialized in modules/lookup_ldap.c:lookup_init()
which can cause it to be incorrect leading to a failure to locate a
base dn.
---
 CHANGELOG             |    1 +
 modules/lookup_ldap.c |    1 +
 2 files changed, 2 insertions(+)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -206,6 +206,7 @@
 - fix out of order amd timestamp lookup.
 - fix ldap default schema config.
 - fix ldap default master map name config.
+- fix map format init in lookup_init().
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/modules/lookup_ldap.c
+++ autofs-5.0.5/modules/lookup_ldap.c
@@ -1648,6 +1648,7 @@ int lookup_init(const char *mapfmt, int
 	/* If a map type isn't explicitly given, parse it like sun entries. */
 	if (mapfmt == NULL)
 		mapfmt = MAPFMT_DEFAULT;
+	is_amd_format = 0;
 	if (!strcmp(mapfmt, "amd")) {
 		is_amd_format = 1;
 		ctxt->format = MAP_FLAG_FORMAT_AMD;
